xml/System.Buffers.Text/Base64Url.xml

```diff
@@ -373,7 +373,7 @@
         <param name="source">The input span which contains binary data that needs to be encoded.</param>
         <summary>Encodes the span of binary data into unicode ASCII chars represented as Base64Url.</summary>
         <returns>A char array which contains the result of the operation, i.e. the ASCII chars in Base64Url.</returns>
-        <remarks>This implementation of the base64url encoding omits the optional padding characters.</remarks>
+        <remarks>This implementation of the base64url encoding omits the optional padding characters. The encoding uses '-' and '_' instead of the '+' and '/' characters used in standard Base64 encoding.</remarks>
```

Comment:
There's inconsistency in capitalization of base64url, e.g. here and on line 375.